src/SFA.DAS.LevyTransferMatching.Infrastructure/Dto/AccountDto.cs

```diff
@@ -2,7 +2,8 @@
 {
     public class AccountDto
     {
+        public string EncodedAccountId { get; set; }
         public string DasAccountName { get; set; }
-        public decimal RemainingTransferAllowance { get; set; }
+        public decimal? RemainingTransferAllowance { get; set; }
```

Comment:
I belive this is because `AccountDto` is used by both the `UserService` (in getting accounts for a logged in user, and therefore it doesn't have the `RemainingTransferAllowance` property), and `AccountService` for getting the detail account (this time including `RemainingTransferAllowance`).

I can set it to become not-nullable if you like, I just figured this would be neater. It's clear that if the property is populated that the value was supplied as part of the call.

Let me know if you still want this changing.

Reply:
Alternatively, I can make this non-nullable, and create a brand new model, e.g. `UserAccountDto`, which doesn't have this property. What would you prefer?

Reply:
I think the latter, a new model, is better, and not have that property on it.

Reply:
Agreed on subsequent discussion to review request/response models for services as a whole as tech debt.

Reply:
I agree, but if we're making changes to existing types, which we will then have to unpick, I think it'd be better to just create a new dto now.

Reply:
This has been addressed, along with the API concern.